src/test/scala/it/unibo/authsim/library/dsl/attack/builders/DictionaryAttackOptionalBuilderTest.scala

```diff
@@ -11,7 +11,7 @@ import org.scalatestplus.mockito.MockitoSugar.mock
 
 import scala.concurrent.duration.Duration
 
-class DictionaryAttackBuilderTest extends AnyWordSpec:
+class DictionaryAttackOptionalBuilderTest extends AnyWordSpec:
```

Comment:
Also this class has been renamed, why?

Reply:
See the previous comment, ah it is probably due to the fixes I made when I introduce the mock class for the Algorithm, or I did a rename that I do not remember and I accidentally rename also your classes.
NP which name should I use?
` DictionaryAttackBuilderTest` is not it? and `BruteForceAttackBuilderTest  `